addon/components/new-bourbon-select-field.js

```diff
@@ -39,6 +38,10 @@ export default Component.extend({
   },
 
   didInsertElement() {
+    if (this.get('lazyLoad')) {
+      this.set('loadOptions', false);
```

Comment:
Since `loadOptions` defaults to `true` above (line 26), is there ever a time when the content (options list) is rendered before `loadOptions` is set to `false` here? Just making sure we're not loading/unloading unnecessarily.

If the content is being loaded and immediately unloaded, it might be better to default `loadOptions` to `false`, and then in here set it to `true` if `lazyLoad` is `false`. This way `loadOptions` is never `true` unless we mean to actually load the content.